### PR TITLE
build: update dependencies for CVE-2023-2253

### PR DESCRIPTION
**Pull request by pgcrooks-sysdig**
Signed-off-by: Paul Crooks <paul.crooks@sysdig.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area build

> /area tests

> /area proposals

**What this PR does / why we need it**:
Update dependencies to address open CVEs.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: runtimes/cloudformation/go.mod [hunk 1]
```diff
@@ -3,42 +3,42 @@ module github.com/falcosecurity/kilt/runtimes/cloudformation
...
 
 require (
-	github.com/containerd/stargz-snapshotter/estargz v0.12.1 // indirect
+	github.com/containerd/stargz-snapshotter/estargz v0.14.3 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/docker/cli v20.10.20+incompatible // indirect
-	github.com/docker/distribution v2.8.1+incompatible // indirect
-	github.com/docker/docker v20.10.24+incompatible // indirect
+	github.com/docker/cli v24.0.2+incompatible // indirect
+	github.com/docker/distribution v2.8.2+incompatible // indirect
```
Comment (gnosek) [inline runtimes/cloudformation/go.mod:20]: I suppose this is the important line? ;)

Comment (pgcrooks-sysdig) [reply runtimes/cloudformation/go.mod:20]: I sure hope so!

**Pull request merged**